common/changes/@uifabric/styling/keyou-add-unregister-icon-export_2018-07-23-18-24.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/styling",
+      "comment": "Add unregisterIcons to styling export",
+      "type": "minor"
```

Comment:
this is a minor change. (when you add functionality, it's a minor.)

Good test to consider mentally: if you publish this, start using it, and then go back a minor build, will it break?

Reply:
Yeah I wasn't sure...thanks